### PR TITLE
WildFire - py3 version

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
In progress - deprecation of old integration and strings fixes from product writer.

## Related Issues
fixes: https://github.com/demisto/etc/issues/16548
fixes: https://github.com/demisto/etc/issues/16413
reference:  https://github.com/demisto/etc/issues/16548 - only the command. playbook will be done in a second PR.
## Description
addition of hash_list arg in get-verdicts
addition of upload-url
deprecating command from JS integrations

## Required version of Demisto
4.0.0

## Does it break backward compatibility?
   - No - compatible with JS version, passes it's test playbook

## Must have
- [x] Tests - also unit tests
- [x] Documentation - https://github.com/demisto/etc/issues/16769
- [ ] Code Review

### CODE REVIEW

**Comment by bakatzir** [comment]
@yuvalbenshalom  michal and I are looking at design for the URL and File-URL detonation playbooks. will be done in a separate issue after finishing design.

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 1]
```diff
@@ -0,0 +1,722 @@
...
+    upload_file_url_uri = URL + URL_DICT["upload_file_url"]
+    body = '''--upload_boundry
+Content-Disposition: form-data; name="apikey"
+
+{apikey}
+--upload_boundry
+Content-Disposition: form-data; name="url"
+
+{url}
+--upload_boundry--'''.format(apikey=TOKEN, url=upload)
```
Comment (yuvalbenshalom) [inline Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: better fusee py3 formatting style `f'my string with {my_val}'`

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+        file_hash, report, file_info = wildfire_get_report(element)
+        create_report(file_hash, report, file_info, format_, verbose)
+
+
+def wildfire_file_command():
+    inputs = file_args_handler(demisto.args().get('file'), demisto.args().get('md5'), demisto.args().get('sha256'))
+    for element in inputs:
+        if sha1Regex.match(element):
+            demisto.results({
+                'Type': 11,
```
Comment (yuvalbenshalom) [inline Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: what is type 11?

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 3]
```diff
@@ -0,0 +1,722 @@
...
+        if url.find('/get/sample') != -1:
+            demisto.results(
+                'Sample was not found. '
+                'Please note that grayware and benign samples are available for 14 days only. '
+                'For more info contact your wildfire representative.')
+            sys.exit(0)
+        # report not found
+        if url.find('/get/report') != -1:
+            demisto.results('Report not found.')
+            sys.exit(0)
```
[the remark this answers is not in the document]
Comment (bakatzir) [reply Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: I do not want to return an entry of type error

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 4]
```diff
@@ -0,0 +1,722 @@
...
+    )
+
+    if str(result.reason) == 'Not Found':
+        # sample not found
+        if url.find('/get/sample') != -1:
+            demisto.results(
+                'Sample was not found. '
+                'Please note that grayware and benign samples are available for 14 days only. '
+                'For more info contact your wildfire representative.')
+            sys.exit(0)
```
[the remark this answers is not in the document]
Comment (bakatzir) [reply Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: I do not want to return an entry of type error

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 5]
```diff
@@ -0,0 +1,722 @@
...
+        # report not found
+        if url.find('/get/report') != -1:
+            demisto.results('Report not found.')
+            sys.exit(0)
+
+    if result.status_code < 200 or result.status_code >= 300:
+        if result.status_code in ERROR_DICT:
+            return_error('Request Failed with status: {status} Reason is: {reason}'.format(
```
[the remark this answers is not in the document]
Comment (bakatzir) [reply Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: yes, the reason should be prettified

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 6]
```diff
@@ -0,0 +1,722 @@
...
+        pretty_upload["URL"] = upload_body["url"]
+
+    return pretty_upload
+
+
+def prettify_report_entry(file_info):
+    pretty_report = {
+        'MD5': file_info["md5"],
+        'SHA256': file_info["sha256"],
+        'Status': 'Completed'
```
[the remark this answers is not in the document]
Comment (bakatzir) [reply Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: yes

File: Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py (hunk 2 above)
Comment (bakatzir) [reply Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py]: warning

**Pull request merged**